src/compile/model.ts

```diff
@@ -15,6 +15,12 @@ import {VgData, VgMarkGroup, VgScale, VgAxis, VgLegend} from '../vega.schema';
 import {DataComponent} from './data/data';
 import {LayoutComponent} from './layout';
 import {ScaleComponents} from './scale';
+/* tslint:disable:no-unused-variable */
```

Comment:
Do you really need this set of hacks?  It's weird that tslint isn't friendly with .d.ts export.